hypertrace-graphql-gateway-service-metric-utils/src/main/java/org/hypertrace/graphql/utils/metrics/gateway/MetricAggregationConverter.java

```diff
@@ -18,6 +20,10 @@
 
   @Override
   public Single<MetricAggregation> convert(Value value) {
+    if (Value.getDefaultInstance().equals(value)) {
+      return Single.just(new ConvertedMetricAggregation(null));
```

Comment:
no passing of nulls please - instead, add an explicit static like `ConvertedMetricAggregation.EMPTY`;

Reply:
And, what would the static value correspond to?

Would it look like? `static final ConvertedMetricAggregation EMPTY = new ConvertedMetricAggregation(null);`

Reply:
yep - makes this code more readable and keeps the null inside that class as an implementation detail rather than the caller knowing it's ok to pass null.